Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/Use_Case_1_Two_qubit_RB_without_baking/README.md

```diff
@@ -0,0 +1,138 @@
+# Two-qubit RB generated by cross-resonance CNOTs without pre-baking
+
+_Authors and experimenters: Ziwen Huang, Paul Heidler, Nicholas Bornman_
+
+*Important note: the code in this folder is the code that was used for performing two-qubit randomized benchmarking with cross-resonance*
+*gates for a pair of coupled transmon qubits.*
+
+The file for direct use is [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py), which relies on random Cliffords generation 
+coded in [TwoQ_RB_Sequence_Generation_CNOT_CR.py](TwoQ_RB_Sequence_Generation_CNOT_CR.py).
+
+## 1. Pre-requisite calibrations and some notes
+
+* Calibrate readout pulses and optimize state discrimination.
+* Calibrate and optimize qubit XY pulses. Single qubit operations are based on x90, -x90, x180, y90, -y90, and y180.
+* Our two qubit RB is based on CNOT as the entangling operation which is implemented by direct CR gate. Replace the pertinent lines in the definition of the function `play_sequence` in [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py) (switch case 49) if echoed CR is to be used.
+* No baking will be directly called by the user. The user can easily customize their gates, especially the two-qubit gate pulses.
+* Running gate sequences other than those for RB is also allowed if one manually set the `sequence_list`. (See ##### TEST EXAMPLE in [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py).)
+
+
+
+## 2. [The configuration](configuration.py)
+
+
+### 2.1 The elements
+The configuration consists of 4 elements:
+* `rr1` and `rr2` send readout pulses and measure the transmitted signal for each qubit
+* `q1_xy` and `q2_xy` send drive XY pulses to each qubit
+* `cr_drive_c1t2`, `cr_drive_c2t1` send cross-resonance drive to the control qubit (at target frequency)
+* `cr_cancel_c1t2`, and `cr_cancel_c2t1` send cancellation drive the target qubit (also at target frequency)
```

Comment:
Let's add - In cr_cancel_c1t2, qubit 1 acts as the control qubit, and qubit 2 as the target. Conversely, in cr_cancel_c2t1, qubit 2 is the control qubit, and qubit 1 is the target.